### PR TITLE
Update how-to-get-key-by-prefix doc.

### PR DESCRIPTION
**Pull request by fuweid**
Fixes: #797

### CODE REVIEW

**Comment by fuweid** [comment]
cc @celestehorgan

**Review by celestehorgan — CHANGES_REQUESTED** [review]

File: content/en/docs/v3.5/tutorials/how-to-get-key-by-prefix.md [hunk 1]
```diff
@@ -4,6 +4,32 @@ description: Guide to extracting etcd keys by their prefix
 weight: 300
 ---
 
+## Pre-requisites
+
+* [Install tools](https://etcd.io/docs/v3.5/install/)
```
[the remark this answers is not in the document]
Comment (fuweid) [reply content/en/docs/v3.5/tutorials/how-to-get-key-by-prefix.md]: Updated! PTAL

File: content/en/docs/v3.5/tutorials/how-to-get-key-by-prefix.md [hunk 2]
```diff
@@ -4,6 +4,32 @@ description: Guide to extracting etcd keys by their prefix
...
+
+### Options
+
+```bash
+--prefix, get a range of keys with matching prefix
+```
+
+### Example
+
+
```
Comment (jmhbnz) [inline content/en/docs/v3.5/tutorials/how-to-get-key-by-prefix.md:32]: nit: MD012/no-multiple-blanks: Multiple consecutive blank lines [Expected: 1; Actual: 2]markdownlint

**Review by jmhbnz — APPROVED** [review]
LGTM - Minor nit below if you have time.

File: content/en/docs/v3.5/tutorials/how-to-get-key-by-prefix.md (hunk 2 above)
Comment (fuweid) [reply content/en/docs/v3.5/tutorials/how-to-get-key-by-prefix.md]: Done! thanks

File: content/en/docs/v3.5/tutorials/how-to-get-key-by-prefix.md [hunk 3]
```diff
@@ -4,6 +4,31 @@ description: Guide to extracting etcd keys by their prefix
 weight: 300
 ---
 
+## Pre-requisites
+
+* [Install etcdctl](https://etcd.io/docs/v3.5/install/)
```
Comment (khanhtc1202) [inline content/en/docs/v3.5/tutorials/how-to-get-key-by-prefix.md:9]:
```suggestion
* Install [etcdctl](https://etcd.io/docs/v3.5/install/)
```
nits

File: content/en/docs/v3.5/tutorials/how-to-get-key-by-prefix.md [hunk 4]
```diff
@@ -4,6 +4,31 @@ description: Guide to extracting etcd keys by their prefix
 weight: 300
 ---
 
+## Pre-requisites
```
Comment (khanhtc1202) [inline content/en/docs/v3.5/tutorials/how-to-get-key-by-prefix.md:7]:
```suggestion
## Prerequisites
```
nits

File: content/en/docs/v3.5/tutorials/how-to-get-key-by-prefix.md (hunk 3 above)
Comment (fuweid) [reply content/en/docs/v3.5/tutorials/how-to-get-key-by-prefix.md:9]: I think etcdctl is about what is etcdctl. [install etcdctl] is about how to get it.
I don't think we should just highlight the etcdctl.

**Pull request merged**